### PR TITLE
Submit button on password-protected link disabled when password field is empty

### PR DESCRIPTION
**Pull request by gmurayama**
#20177

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @PVince81

**Comment by MorrisJobke** [comment]
I just noticed that you opened this PR against stable8.2 - Our development workflow is instead following:

- bring the fix/feature to master (current developing base that will result in the next version)
- if this is a fix we will discuss about the impact and then backport to the stable branches.

**Comment by MorrisJobke** [comment]
I just tested your two commits on top of master -> the whole password field is disabled and I can't type in my password at all.


Can I ask you to open a new PR against master (the base can't be switched after the PR is created)?

File: apps/files_sharing/js/authenticate.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+ $(document).ready(function(){
+    $('#password').keyup(function(){
```
Comment (PVince81) [inline apps/files_sharing/js/authenticate.js]: keyup is not enough, you might need to use `.on('keyup input change', function() { ...` instead

**Comment by gmurayama** [comment]
So sorry, I made a HUGE mistake. I set the password field disabled, not the submit button. Already fixed. I'm going to open a new PR as requested!

**Pull request closed without merging**